src/util/modules.py

```diff
@@ -486,8 +486,19 @@ def force_install_module(cr, module, if_installed=None):
     :param str module: name of the module to install
     :param list(str) or None if_installed: only force the install when these modules are
                                            already installed
-    :return str: the *original* state of the module
```

Comment:
Even if *we* don't use it, that's unfortunate that the API change.

Reply:
are you saying we should not have this change?
if that's the case then adding the assertion `_assert_modules_exists` to the existing implementation alone will force us to use the auto_discovery api for new modules